app/controllers/issues_controller.rb

```diff
@@ -1,30 +1,58 @@
 class IssuesController < ApplicationController
-  before_action :logged_in_user, only: [:create, :destroy]
-  before_action :correct_user,   only: :destroy
-
+  before_action :verify_user_has_logged_in, only: [:new, :create, :destroy]
```

Comment:
It is like that, 'verify_user_has_logged_in' method is defined in the base class 'ApplicationController'